deploy/crds/applicationmonitoring.integreatly.org_applicationmonitorings_crd.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: applicationmonitorings.applicationmonitoring.integreatly.org
+spec:
+  group: applicationmonitoring.integreatly.org
+  names:
+    kind: ApplicationMonitoring
+    listKind: ApplicationMonitoringList
+    plural: applicationmonitorings
+    singular: applicationmonitoring
+  scope: ""
```

Comment:
A Kubebuilder marker should be added to your types to populate this correctly - `+kubebuilder:resource:path=<resource>,scope=Namespaced`